core/templates/pages/contributor-dashboard-page/translation-language-selector/translation-language-selector.component.spec.ts

```diff
@@ -183,6 +188,47 @@ describe('Translation language selector', () => {
     expect(component.activeLanguageCode).toBe('en');
   });
 
+  it('should display the preferred language when the preferred' +
+    ' language is defined', async(() => {
+    component.activeLanguageCode = null;
+    preferredLanguageCode = 'en';
+    // This throws "TS2532". We need to suppress this error because
+    // "constants.SUPPORTED_AUDIO_LANGUAGES" is always defined. Also, it must
+    // contain description for language code 'en' and if it doesn't
+    // then this test should fail.
+    // @ts-ignore
+    const languageDescription = constants.SUPPORTED_AUDIO_LANGUAGES.find(
+      e => e.id === 'en').description;
```

Comment:
Can we use `LanguageUtilService.getAudioLanguageDescription()` here?

Reply:
@sagangwee Yes we can but it will be implemented like this:
```
const languageDescription = new LanguageUtilService(new BrowserCheckerService(
      new WindowRef)).getAudioLanguageDescription(preferredLanguageCode);
```
For this, we need to import `LanguageUtilService`, `BrowserCheckerSerivice`, and `WindowRef`.

Should I update this?

Reply:
I see. What you have then is probably fine, then. Thanks.